packages/gitbook/src/lib/utils.ts

```diff
@@ -12,14 +12,14 @@ import {
 export function getContentTitle(
     space: Space,
     customization: CustomizationSettings | SiteCustomizationSettings,
-    parent: Site | Collection | null,
+    parent?: Site | Collection | null,
```

Comment:
Why add an optional here?

Reply:
ah! I think this is no longer necessary and was related to old published spaces.